content/en/docs/concepts/security/index.md

```diff
@@ -12,565 +12,543 @@ aliases:
-In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
-
-{{< image width="80%" link="./overview.svg" caption="Istio Security Overview" >}}
-
-The Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization
-and audit (AAA) tools to protect your services and data. The goals of Istio security are:
-
-- **Security by default**: no changes needed for application code and infrastructure
+Breaking down a monolithic application into atomic services offers various
+benefits, including better agility, better scalability and better ability to
+reuse services. However, microservices also have particular security needs:
+
+- To defend against man-in-the-middle attacks, they need traffic encryption.
+- To provide flexible service access control, they need mutual TLS and
+   fine-grained access policies.
+- To determine who did what at what time, they need auditing tools.
+
+Istio Security provides a comprehensive security solution to solve these issues.
+This page gives an overview on how you can use Istio security features to secure
+your services, wherever you run them. In particular, Istio security mitigates
+both insider and external threats against your data, endpoints, communication,
+and platform.
+
+{{< image width="75%"
+    link="./overview.svg"
+    caption="Security overview"
+    >}}
 
-- **Defense in depth**: integrate with existing security systems to provide multiple layers of defense
+The Istio security features provide strong identity, powerful policy,
+transparent TLS encryption, and authentication, authorization and audit (AAA)
+tools to protect your services and data. The goals of Istio security are:
 
-- **Zero-trust network**: build security solutions on untrusted networks
+- Security by default: no changes needed to application code and
+    infrastructure
+- Defense in depth: integrate with existing security systems to provide
+    multiple layers of defense
+- Zero-trust network: build security solutions on distrusted networks
 
-Visit our [Mutual TLS Migration docs](/docs/tasks/security/authentication/mtls-migration/) to start using Istio security features with your deployed services.
-Visit our [Security Tasks](/docs/tasks/security/) for detailed instructions to use the security features.
+Visit our
+[mutual TLS Migration docs](/docs/tasks/security/authentication/mtls-migration/)
+to start using Istio security features with your deployed services. Visit our
+[Security Tasks](/docs/tasks/security/) for detailed
+instructions to use the security features.
 
 ## High-level architecture
 
 Security in Istio involves multiple components:
 
-- **Citadel** for key and certificate management
+- A Certificate Authority (CA) for key and certificate management
+- Istiod distributes to the proxies:
```

Comment:
Istiod needs to be introduced before you can describe it's functions wrt security. That said, isn't Istiod too much an implementation detail for the concepts section? I thought one of the goals of the security concepts rewrite was to remove the architecture stuff (move it to the op guide) and focus instead on the security features. Like we did with traffic management.

Reply:
Due to time constraints, we didn't think we'd have time to do a full rewrite.  Instead we focused on updating the documentation to correctly reflect the 1.5 architecture--and remove a few pieces of extraneous information.  We hope to have a rewrite of the Security Concepts page for 1.6.

Reply:
Fair enough, but I still think we can try to keep this a little higher level. For example, my suggestion to use the generic Istio function (e.g., the config API server) instead of the implementation component name (Istiod) wherever possible.

Reply:
@frankbu In general I agree with you. The issue with using the generic function is that the audience won't necessarily know anything about it. Using Istiod in this case is better, because readers will recognize it from their deployment.